### PR TITLE
Changes logging for libraries

### PR DESCRIPTION
**Pull request by dharmit**

### CODE REVIEW

File: container_pipeline/lib/openshift.py [hunk 1]
```diff
@@ -106,10 +106,10 @@ def wait_for_build_status(self, project, build_id, status,
             if current_status == status:
                 return True
             elif not current_status:
-                self.logger.debug('Failed to fetch build status for: {}/{}'
+                self.logger.warning('Failed to fetch build status for: {}/{}'
                                   .format(project, build_id))
                 if empty_retry_count == empty_retries:
-                    self.logger.debug(
+                    self.logger.warn(
```
Comment (bamachrn) [inline container_pipeline/lib/openshift.py]: its logger.warning right?

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
#dotests

**Comment by navidshaikh** [comment]
#dotests

**Pull request merged**